Server/src/services/tools/manage_ugui.py

```
@@ -0,0 +1,104 @@
+    alignment: Annotated[str, "Text alignment (e.g., UpperLeft, MiddleCenter, LowerRight)."] | None = None,
+    raycastTarget: Annotated[bool, "Whether the element blocks raycasts."] | None = None,
+    preserveAspect: Annotated[bool, "Whether the image preserves its aspect ratio."] | None = None,
+
+) -> dict[str, Any]:
+    unity_instance = await get_unity_instance_from_context(ctx)
+
+    params_dict: dict[str, Any] = {
+        "action": action,
+    }
+
+    if type: params_dict["type"] = type
```

Ruff (0.15.7)

[error] Multiple statements on one line (colon)

(E701)